SConstruct

```diff
@@ -28,10 +28,15 @@ def add_sources(sources, directory):
             sources.append(directory + '/' + file)
 
 
+if platform == "osx":
```

Comment:
This is at the toplevel. Which is fine and works too.

This whole file needs a really heavy refactoring anyway 😄 

I guess we can keep it that way for now, but this whole thing needs some love, it's all over the place 😄 

Will merge for now